smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/base/WeldTestBase.java

```diff
@@ -119,10 +119,15 @@ public HealthCenter getHealth() {
         return container.getBeanManager().createInstance().select(HealthCenter.class).get();
     }
 
+    public boolean isStarted() {
+        return getHealth().getStartup().isOk();
+    }
+
     public boolean isReady() {
         System.out
-                .println(getHealth().getReadiness().getChannels().stream().map(ci -> ci.getChannel() + " " + ci.isOk()).collect(
-                        Collectors.joining()));
+                .println(getHealth().getReadiness().getChannels().stream()
+                        .map(ci -> ci.getChannel() + " " + ci.isOk() + " " + ci.getMessage())
+                        .collect(Collectors.joining()));
```

Comment:
At some point, we should probably remove this? :-)

Reply:
It was very useful during tests :)

Reply:
I'm sure it is, but I'm also pretty sure it is annoying otherwise :-) We could make it a trace log.